package.json

```diff
@@ -176,7 +176,7 @@
     "builder-util-runtime": "^9.0.3",
     "cross-env": "5.2.0",
     "del": "3.0.0",
-    "electron": "^19.0.10",
+    "electron": "^20.1.3",
```

Comment:
Please change "version" field to 20.0.0 (line 4)

Reply:
Should I make it a solid version of 20.0.0 or it will just be ok with the @latest?
@sbenmoussati

Reply:
solid version "20.0.0" is ok @NguyenTranHoangSym